src/app/store/reboot-info/reboot-info.selectors.ts

```diff
@@ -0,0 +1,21 @@
+import { createFeatureSelector, createSelector } from '@ngrx/store';
+import { RebootInfoState } from 'app/store/reboot-info/reboot-info.reducer';
+
+export const rebootInfoStateKey = 'rebootInfo';
+
+export const selectRebootInfoState = createFeatureSelector<RebootInfoState>(rebootInfoStateKey);
+
+export const selectRebootInfo = createSelector(
+  selectRebootInfoState,
+  (state) => state,
+);
+
+export const selectThisNodeInfo = createSelector(
```

Comment:
Here as well. Let's add `reboot` keyword in the variable name so it's clear it's only the node specific reboot info.